src/pagination/pagination.ts

```diff
@@ -68,6 +78,19 @@ export class NgbPagination implements OnChanges {
   @Input() ellipses: boolean;
 
   /**
+   * Text to display for the first page button.
+   */
+  @Input()
+  set firstText(value: string) {
+    this._firstText = value;
+    if (value !== this._defaultConfig.firstText) {
+      this._firstTextSr = value;
```

Comment:
In its default configuration, the pagination shows the text (First, Previous, etc.) for screen-readers and the arrows for the other types of displays. This info is not transparent though in the API docs - maybe a potential for enhancement here. 

This difference between sr and non-sr displays was there before and it feels to me like it makes sense to continue to keep it.